packages/governance/src/contractHelper.js

```diff
@@ -99,13 +99,18 @@ const facetHelpers = (zcf, paramManager) => {
    * @param {Record<string, (...args: any[]) => any>} governedApis
    * @returns {GovernedCreatorFacet<CF>}
    */
+
+  /**
+   * @param {{}} originalCreatorFacet
+   * @param {{}} governedApis
```

Comment:
Looks like this is optional:
```suggestion
   * @param {{}} [governedApis]
```

Reply:
done